RecoJets/JetProducers/python/PileupJetID_cfi.py

```diff
@@ -33,7 +35,8 @@
      usePuppi = cms.bool(False),
 #     residualsTxt     = cms.FileInPath("RecoJets/JetProducers/data/download.url") # must be an existing file
 )
-run2_miniAOD_UL.toModify(pileupJetId, algos = _chsalgos_106X_UL17)
+run2_miniAOD_UL.toModify(pileupJetId, algos = _chsalgos_106X_UL18)
+(run2_miniAOD_UL & run2_jme_2017).toModify(pileupJetId, algos = _chsalgos_106X_UL17)
```

Comment:
```suggestion
run2_miniAOD_UL.toModify(pileupJetId, algos = _chsalgos_106X_UL17)
(run2_miniAOD_devel & run2_jme_2018).toModify(pileupJetId, algos = _chsalgos_106X_UL18)
```
is what I understood is needed
- the first line reverts back to the current default of the PR
- the second enables the 2018 PU ID for the 2018 "devel" re-miniAOD

Reply:
there is no `run2_jme_2018_cff`, did you mean `run2_miniAOD_devel.toModify(pileupJetId, algos = _chsalgos_106X_UL18)` for second line?

Reply:
my expectation is that my proposed change above  when used with `--era Run2_2018,run2_miniAOD_devel --procModifiers run2_miniAOD_UL` will give an updated UL re-mini with the 2018 PU ID for the 2018 era/setup. (to compare with the production default --era Run2_2018 --procModifiers run2_miniAOD_UL` which is locked to use PU ID 2017 by the no-change policy due to the 2018 campaign being already in progress)

Reply:
> there is no `run2_jme_2018_cff`, did you mean `run2_miniAOD_devel.toModify(pileupJetId, algos = _chsalgos_106X_UL18)` for second line?

Ehm, it may be better that `run2_jme_2018` actually existed.
Otherwise, the last line would have `(run2_miniAOD_devel (~run2_jme_2016) & (~run2_jme_2017)).`